packages/core/test/feedback.test.ts

```diff
@@ -261,10 +261,13 @@ describe('captureFeedback', () => {
 
     expect(typeof eventId).toBe('string');
 
-    expect(mockTransport).toHaveBeenCalledWith([
+    const [[envelope]] = mockTransport.mock.calls;
```

Comment:
I don't understand the change, the original code should do the same check.

````suggestion
    expect(mockTransport).toHaveBeenCalledWith([
````

Reply:
True, it wasn't before, but now it is, so I roll back the original implementation here 
  https://github.com/getsentry/sentry-react-native/pull/4568/commits/45ee6fadd66e5d745627d4eccdbab5698cbfb0c7